popup-portfolio.css

```diff
@@ -0,0 +1,293 @@
+.popuppt {
+  background-color: #c1c7d0;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 9;
+  overflow: auto;
+  padding: 16px;
+  opacity: 1;
+}
+
+.projectcard {
+  background-color: #fff;
+  width: 100%;
+  height: 55rem;
+  border-radius: 10px;
+  padding: 12px;
+  opacity: 1;
+}
+
+.imagepopup {
+  position: absolute;
+  margin-bottom: 0.75rem;
+  z-index: 1;
+  width: 21.9rem;
+}
+
+.closingbutton {
+  position: absolute;
+  z-index: 1;
+  margin-left: 19.17rem;
+  margin-top: 1.15rem;
+  color: #67798e;
+  font-size: 24px;
+}
+
+.closingbutton:hover {
+  cursor: pointer;
+}
+
+.titleproject {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  margin-top: 17rem;
+  margin-bottom: 1.25rem;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  width: 16.2rem;
+  height: 8.45rem;
+  color: #172b4d;
+}
+
+.leftblock {
+  display: flex;
+  width: 20rem;
+  margin-bottom: 2rem;
+}
+
+.techs {
+  width: 100%;
+  list-style: none;
+  display: flex;
+  gap: 0.5rem;
+}
+
+.liproject {
+  padding: 0.5rem 0.75rem;
+  background: #ebf0ee;
+  border-radius: 0.25rem;
+  width: fit-content;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 0.75rem;
+  font-weight: 600;
+  line-height: 1rem;
+}
+
+.paragraph {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  color: #344563;
+  margin-bottom: 1rem;
+}
+
+.bottomblock {
+  width: 100%;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.sourcebuttonport {
+  padding: 0.75rem 1rem;
+  gap: 0.75rem;
+  background: #36b37f;
+  border: none;
+  border-radius: 8px;
+  width: 10rem;
+  height: 3rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+}
+
+.livebuttonport {
+  padding: 0.75rem 1rem;
+  gap: 0.75rem;
+  background: #36b37f;
+  border: none;
+  border-radius: 8px;
+  width: 9rem;
+  height: 3rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+}
+
+.closingpopup {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .popuppt {
+    background-color: #c1c7d0;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    z-index: 9;
+    overflow: auto;
+    padding: 16px;
+    opacity: 1;
+  }
+
+  .projectcard {
+    background-color: #fff;
+    width: 80%;
+    height: 100%;
+    border-radius: 10px;
+    padding: 12px;
+    opacity: 1;
+    margin-left: 10%;
+    margin-right: 10%;
+  }
```

Comment:
- [ OPTIONAL ] In your desktop view, there is some text overflowing. I would suggest that you apply `overflow: scroll` on the project card container so your text stays in the container :+1: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/217715624-cf5f7c8a-0845-4892-87a7-669d7dfe1038.png)